lib/menu.rb

```diff
@@ -0,0 +1,20 @@
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    # I could have the menu read from a file in a private method
+    # at initialisation. I have chosen to keep things simple here for
+    # ease of testing
+    @dishes = { burrito: '6.80' }
+  end
+
+  def view
```

Comment:
The user may find it more useful if they are provided with a list that provides the available dishes and prices. This can be achieved by using a block.